speechmatics/models.py

```diff
@@ -131,6 +131,9 @@ def asdict(self) -> Dict[Any, Any]:
     audio_filtering_config: dict = None
     """Configuration for limiting the transcription of quiet audio."""
 
+    transcript_filtering_config: dict = None
```

Comment:
The correct type for this is `Optional[dict]` since None isn't an instance of `dict`. (I am aware this is wrong above as well)

Reply:
Thanks for pointing that out! Will fix both, and I'm also going to ensure that this change works correctly with my container test before merging.

Reply:
Unfortunately, @HennerM, if I change either of these I get many test errors about type "_UnionGenericAlias is not JSON serializable" so I think it might be beyond the scope of this change to fix this.

Reply:
What type did you use? It should be `from typing import Optional`

Reply:
Yep, that's already been imported. I changed the lines to:
```
    audio_filtering_config: dict = Optional[dict]
    transcript_filtering_config: dict = Optional[dict]
```
The output of make test is:
```
==================================================================== short test summary info =====================================================================
ERROR tests/test_client.py::test_batch_mock_jobs - AssertionError: The following responses are mocked but not requested:
FAILED tests/test_cli.py::test_rt_main_with_basic_options - TypeError: Object of type _UnionGenericAlias is not JSON serializable
FAILED tests/test_cli.py::test_rt_main_with_temp_token_option - TypeError: Object of type _UnionGenericAlias is not JSON serializable
FAILED tests/test_cli.py::test_rt_main_with_toml_config - TypeError: Object of type _UnionGenericAlias is not JSON serializable
FAILED tests/test_cli.py::test_rt_main_with_all_options - TypeError: Object of type _UnionGenericAlias is not JSON serializable
FAILED tests/test_cli.py::test_rt_main_with_config_file - TypeError: Object of type _UnionGenericAlias is not JSON serializable
FAILED tests/test_cli.py::test_rt_main_with_config_file_cmdline_override - TypeError: Object of type _UnionGenericAlias is not JSON serializable
FAILED tests/test_client.py::test_handlers_called - TypeError: Object of type _UnionGenericAlias is not JSON serializable
FAILED tests/test_client.py::test_middlewares_called - TypeError: Object of type _UnionGenericAlias is not JSON serializable
FAILED tests/test_client.py::test_force_end_session_from_event_handler - TypeError: Object of type _UnionGenericAlias is not JSON serializable
FAILED tests/test_client.py::test_force_end_session_from_middleware[AddAudio] - TypeError: Object of type _UnionGenericAlias is not JSON serializable
FAILED tests/test_client.py::test_update_transcription_config_sends_set_recognition_config - TypeError: Object of type _UnionGenericAlias is not JSON serializable
FAILED tests/test_client.py::test_start_recognition_sends_speaker_diarization_config - TypeError: Object of type _UnionGenericAlias is not JSON serializable
FAILED tests/test_client.py::test_client_stops_when_asked_and_sends_end_of_stream - TypeError: Object of type _UnionGenericAlias is not JSON serializable
FAILED tests/test_client.py::test_language_pack_info_is_stored - TypeError: Object of type _UnionGenericAlias is not JSON serializable
FAILED tests/test_client.py::test_batch_mock_jobs - TypeError: Object of type _UnionGenericAlias is not JSON serializable
FAILED tests/test_models.py::test_batchtranscriptionconfig_json_simple - TypeError: Object of type _UnionGenericAlias is not JSON serializable
```

Reply:
```suggestion
    transcript_filtering_config: Optional[dict] = None
```

the change needs to be in the type annotation, not in the assignment

Reply:
Ah got it, thanks! Changed that now.